### PR TITLE
Readme Improvements

### PR DESCRIPTION
**Pull request by QuintinWillison**
The primary purpose of this pull request is to add some links to resources that developers adopting our Flutter plugin might find useful on their integration journey.

I've also used this as an opportunity to make some minor structural changes as well as some grammatical improvements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,13 +23,20 @@ or visit our
 [Support and Help](https://www.ably.io/support)
 site to discuss privately.
 
+## Resources
+
+- [Quickstart Guide](https://www.ably.io/documentation/quick-start-guide?lang=flutter)
+- [Introducing the Ably Flutter plugin](https://www.ably.io/blog/ably-flutter-plugin) by [Srushtika](https://github.com/Srushtika) (Developer Advocate)
+- [Building a Realtime Cryptocurrency App with Flutter](https://www.ably.io/tutorials/realtime-cryptocurrency-app-flutter) by [pr-Mais](https://github.com/pr-Mais) and [escamoteur](https://github.com/escamoteur)
+- [Building realtime apps with Flutter and WebSockets: client-side considerations]()
```
Comment (ramiro-nd) [inline README.md:31]: Link is missing, should be:

[Building realtime apps with Flutter and WebSockets: client-side considerations](https://www.ably.io/topic/websockets-flutter)

**Review by ramiro-nd — CHANGES_REQUESTED** [review]
https://github.com/ably/ably-flutter/pull/79/files#r565223758

File: README.md (hunk 1 above)
Comment (QuintinWillison) [reply README.md:31]: Good spot. Sorry, I didn't test every link - I must have been distracted when I wrote that bullet. Will fix. 😁

Comment (QuintinWillison) [reply README.md]: Fixed: https://github.com/ably/ably-flutter/pull/79/commits/97dcce9cd4fd75036f0e35fdf91a4128089b29a0

**Review by ramiro-nd — APPROVED** [review]

**Pull request merged**